.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "0.2.0",
```

Comment:
@shahbozkhurramov .vscode folder should be deleted. Why is it added? Do we have gitignore?

Reply:
Yes I have gitignore file. But how can I handle this problem?

Reply:
Just forget for now.